judge/views/organization.py

```diff
@@ -80,6 +81,16 @@ def get_context_data(self, **kwargs):
         context = super(OrganizationHome, self).get_context_data(**kwargs)
         context['title'] = self.object.name
         context['can_edit'] = self.can_edit_organization()
+        context['is_member'] = self.request.profile in self.object if self.request.user.is_authenticated else False
+        context['new_problems'] = Problem.objects.filter(is_public=True, is_organization_private=True, organizations=self.object) \
+                                                 .order_by('-date', '-id')[:7]
```

Comment:
Can you turn this `7` into a constant called `DMOJ_NEW_PROBLEM_COUNT`, or something to that effect?